src/storage/responses/choice.get.json

```diff
@@ -0,0 +1,7 @@
+{
+    "data": {
+        "id": 3,
+        "message": "LeroiJenkins",
```

Comment:
It's Leeroy Jenkins
https://www.youtube.com/watch?v=mLyOj_QD4a4

Reply:
haha sorry it's fixed